apps/core-app/src/main/utils/common-util.test.ts

```
@@ -0,0 +1,42 @@
+const tempRoots: string[] = []
+
+function createTempRoot(): string {
+  const root = fs.mkdtempSync(path.join(os.tmpdir(), 'tuff-common-util-'))
+  tempRoots.push(root)
+  return root
+}
+
+afterEach(() => {
+  while (tempRoots.length > 0) {
+    const root = tempRoots.pop()
+    if (!root) continue
```

ESLint

[error] Expect newline after if

(antfu/if-newline)